openlibrary/plugins/openlibrary/js/edit.js

```diff
@@ -235,15 +235,28 @@ export function validateIdentifiers(data) {
 export function initClassificationValidation() {
     initJqueryRepeat();
     const dataConfig = JSON.parse(document.querySelector('#classifications').dataset.config);
+
+    // Prevent form submission on Enter for classification fields
+    $('#select-classification, #classification-value').on('keydown', function(e) {
```

Comment:
On my testing in firefox/chrome, having the select in focus while pressing enter had no effect, so we don't need to block that case. Also if we do, then pressing enter will trigger add!

```suggestion
    $('#classification-value').on('keydown', function(e) {
```